src/types/markers.js

```diff
@@ -282,25 +282,31 @@ export type InvalidationPayload = {
  * be included depending on what states happen during the load.  Also note
  * that redirects are logged as well.
  */
+
+type NetworkTimeDetails = {
+  domainLookupStart?: Milliseconds,
+  domainLookupEnd?: Milliseconds,
+  connectStart?: Milliseconds,
+  tcpConnectEnd?: Milliseconds,
+  secureConnectionStart?: Milliseconds,
+  connectEnd?: Milliseconds,
+  requestStart?: Milliseconds,
+  responseStart?: Milliseconds,
+  responseEnd?: Milliseconds,
+};
+
 export type NetworkPayload = {
   type: 'Network',
   URI?: string,
   RedirectURI?: string,
   id: number,
   pri: number, // priority of the load; always included as it can change
   count?: number, // Total size of transfer, if any
+  dur: number,
   status: string,
   startTime: Milliseconds,
   endTime: Milliseconds,
-  domainLookupStart?: Milliseconds,
-  domainLookupEnd?: Milliseconds,
-  connectStart?: Milliseconds,
-  tcpConnectEnd?: Milliseconds,
-  secureConnectionStart?: Milliseconds,
-  connectEnd?: Milliseconds,
-  requestStart?: Milliseconds,
-  responseStart?: Milliseconds,
-  responseEnd?: Milliseconds,
+  timing: NetworkTimeDetails | null,
```

Comment:
I believe the timing here isn't a property of the `NetworkPayload`, but a list of properties that are included in the payload. This would use the `...` (object type spread).  I'd be happy to investigate and discuss further, although it sounds like this is still "to be determined."